server/data_common/dataset_metadata.py

```diff
@@ -42,7 +42,7 @@ def extrapolate_dataset_location_from_config(server_config: ServerConfig, datase
         dataset = dataset_explorer_location.pop(-1)
         url_dataroot = "/".join(dataset_explorer_location)
         dataroot = None
-        for key, dataroot_dict in server_config.multi_dataset__dataroot.items():
+        for _, dataroot_dict in server_config.multi_dataset__dataroot.items():
```

Comment:
convention for unused variable

Reply:
nit of a nit: just use `.values()` on the dict?

Reply:
ty ty 🤦‍♂️